contract_forecast/tests/test_contract_line_forecast_period.py

```diff
@@ -98,80 +92,77 @@ def test_forecast_period_on_contract_line_update_5(self):
     def test_forecast_period_on_contract_line_update_6(self):
         self.acct_line.write(
             {
-                'date_start': "{this_year}-01-01".format(
-                    this_year=self.this_year),
-                'recurring_next_date': "{this_year}-01-01".format(
-                    this_year=self.this_year),
-                'date_end': "{this_year}-01-28".format(
-                    this_year=self.this_year),
-                'recurring_rule_type': "monthly",
-                'recurring_invoicing_type': 'pre-paid',
+                "date_start": "{this_year}-01-01".format(this_year=self.this_year),
+                "recurring_next_date": "{this_year}-01-01".format(
+                    this_year=self.this_year
+                ),
+                "date_end": "{this_year}-01-28".format(this_year=self.this_year),
+                "recurring_rule_type": "monthly",
+                "recurring_invoicing_type": "pre-paid",
             }
         )
         self.assertTrue(self.acct_line.forecast_period_ids)
         self.assertEqual(len(self.acct_line.forecast_period_ids), 1)
 
     @mute_logger("odoo.addons.queue_job.models.base")
-    def test_forecast_period_on_contract_line_update_6(self):
+    def test_forecast_period_on_contract_line_update_7(self):
```

Comment:
These tests titles are really a bad convention, maybe give them real names, since you're already renaming them?